Sources/TuistLoader/Models+ManifestMappers/RunAction+ManifestMapper.swift

```diff
@@ -25,6 +25,8 @@ extension TuistGraph.RunAction {
 
         let options = try TuistGraph.RunActionOptions.from(manifest: manifest.options, generatorPaths: generatorPaths)
 
+        let language = manifest.language
```

Comment:
nit: This can be probably used directly, I don't think defining as its own variable brings any real value here